RHEL/7/input/xccdf/system/accounts/session.xml

```diff
@@ -46,12 +46,13 @@ left unattended.
 </description>
 <rationale>
 </rationale>
-<ocil clause="value of TMOUT is not greater than or equal to expected setting">
+<ocil clause="value of TMOUT is not less than or equal to expected setting">
```

Comment:
On 4/15/16 6:36 PM, Martin Preisler wrote:
>
> In RHEL/7/input/xccdf/system/accounts/session.xml 
> <https://github.com/OpenSCAP/scap-security-guide/pull/1212#discussion_r59950619>:
>
> > @@ -46,12 +46,13 @@ left unattended.
> >  </description>
> >  <rationale>
> >  </rationale>
> > -<ocil clause="value of TMOUT is not greater than or equal to expected setting">
> > +<ocil clause="value of TMOUT is not less than or equal to expected setting">
>
> Are you sure about this change? The timeout has to be lower, or "not 
> greater". I think the original clause was correct.
>

With the STIG mandated value being`````TMOUT=600`````:

If the system is configured as TMOUT=400, the timeout window is less 
than the requirement and should pass.
If the system is configured as TMOUT=900, the timeout window is greater 
than the requirement and should fail.

The *original* OCIL read: "If TMOUT is not greater than or equal to 600, 
this is a finding"

That would be incorrect. If TMOUT=900 on the system, and TMOUT=600 by 
policy, this rule should *fail*